portfolio/src/main/webapp/index.html

```diff
@@ -5,18 +5,58 @@
     <title>Lorem</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Poiret+One&display=swap" rel="stylesheet">
   </head>
   <body>
+    <br/>
+    <br/>
+    <ul class="icon" >
+    <li><a class="long-icon"  href="/login"><img src="./images/loginicon.png" height="100" width="200"></a></li>
+    <li><a  href="/journal.html">&nbsp;&nbsp;<img src="./images/journalicon.png" height="120" width="120"></a></li>
+    <li><a  href="/charts.html">&nbsp;&nbsp;<img src="./images/chartsicon.png" height="120" width="120"></a></li>
+    <li><a  href="/resources.html">&nbsp;&nbsp;<img src="./images/resourcesicon.png" height="120" width="120"></a></li>
+    <li><a  href="/index.html">&nbsp;&nbsp;<img src="./images/homeicon.png" height="150" width="120"></a></li>
+    </ul>
+
+    <ul class="lorem-logo">
+    <li>&ensp;L</li>
+    <li>&nbsp;O</li>
+    <li>&ensp;R</li>
+    <li>&ensp;E</li>
+    <li>&nbsp;M</li>
+    </ul>
+    <!-- TODO: Make a description box of feature appear when 
```

Comment:
Instead of duplicating these, can you move them to a shared html file?  Code duplication becomes a maintenance headache, where it's very likely they'll become desynchronized over time